openapi-cli/src/main/java/io/ballerina/openapi/cmd/BallerinaCodeGenerator.java

```diff
@@ -503,9 +506,18 @@ public List<GenSrcFile> generateBallerinaService(Path openAPI, String serviceNam
         }
         this.diagnostics.addAll(serviceGenerationHandler.getDiagnostics());
         this.diagnostics.addAll(TypeHandler.getInstance().getDiagnostics());
+        printDiagnostic(diagnostics);
         return sourceFiles;
     }
 
+    private void printDiagnostic(List<Diagnostic> diagnostics) {
+        if (!diagnostics.isEmpty()) {
+            for (Diagnostic diagnostic : diagnostics) {
+                outStream.println(diagnostic.message());
```

Comment:
i think this would print the line numbers also

Reply:
shall we do like `outStream.println(String.format("%s: %s", diagnostic.diagnosticInfo().severity(), diagnostic.message()));`